x/ccv/provider/keeper/grpc_query_test.go

```diff
@@ -720,14 +722,19 @@ func TestQueryConsumerChains(t *testing.T) {
 			tc.setup(ctx, pk)
 			req := types.QueryConsumerChainsRequest{
 				Phase: tc.phase_filter,
-				Limit: tc.limit,
+				Pagination: &sdkquery.PageRequest{
```

Comment:
Not sure if we test this, but would be nice to have a test that uses the `PageResponse` to construct the new `PageRequest` and verify that everything works as expected.